modules/auxiliary/dos/windows/http/rb_dos_iis_2021_31166.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Dos
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows IIS HTTP Protocol Stack DOS',
+        'Description' => %q{
+          This module is a proof of concept for CVE-2021-31166 ("HTTP Protocol
+          Stack Remote Code Execution Vulnerability") to exploit a DOS attack,
+          a use-after-free dereference in http.sys patched by Microsoft
+          in May 2021. 
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Max @_mxms',                                   # Vulnerability discovery
+          'Stefan Blair @fzzyhd1',                        # Vulnerability discovery
+          'Axel Souchet 0vercl0k',                        # Exploit in the wild
+          'Maurice LAMBERT <mauricelambert434@gmail.com>' # msf module

```

Comment:
Nice job on adding these in, however unfortunately our parser doesn't like this format. Here is the suggested format for this to keep the info in but make our parser happy.

```suggestion
          'Max',                                                                                  # Aka @_mxms. Vulnerability discovery
          'Stefan Blair',                                                                       # Aka @fzzyhd1. Vulnerability discovery
          'Axel Souchet',                                                                    # Aka @0vercl0k. PoC exploit
          'Maurice LAMBERT <mauricelambert434[at]gmail.com>' # msf module
```
Change on your email is for parser purposes as well. Basically we can't do `@` symbols or parser won't like it so replaced that with `[at]` as per https://github.com/rapid7/metasploit-framework/blob/04e8752b9b74cbaad7cb0ea6129c90e3172580a2/modules/payloads/singles/cmd/unix/reverse_awk.rb#L20-L22